controller/comments.go

```diff
@@ -241,6 +241,14 @@ func ConvertComment(request *http.Request, comment comment.Comment, additional .
 			Related: &relatedURL,
 		},
 	}
+	if comment.ParentCommentID.Valid == true {
```

Comment:
Since this PR is modifying the convert function, we should add test cases for List & Show methods. Test case for `Show` should go inside `controller/comments_blackbox_test.go` and test for `List` should go insude `controller/work_item_comments_blackbox_test.go`

Reply:
Ok, added tests for both.